mmv1/products/firebase/api.yaml

```diff
@@ -152,3 +152,61 @@ objects:
           Immutable. The globally unique, Firebase-assigned identifier of the App.
 
           This identifier should be treated as an opaque token, as the data format is not specified.
+  - !ruby/object:Api::Resource
+    name: 'AndroidApp'
+    min_version: beta
+    base_url: projects/{{project}}/androidApps
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Google Cloud Firebase Android application instance
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/'
+      api: 'https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.androidApps'
+    async: !ruby/object:Api::OpAsync
+      actions: ["create"]
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The fully qualified resource name of the App, for example:
+
+          projects/projectId/androidApps/appId
+        output: true
+      - !ruby/object:Api::Type::String
+        name: displayName
+        required: true
+        description: |
+          The user-assigned display name of the App.
+      - !ruby/object:Api::Type::String
+        name: appId
+        output: true
+        description: |
+          Immutable. The globally unique, Firebase-assigned identifier of the App.
+
+          This identifier should be treated as an opaque token, as the data format is not specified.
+      - !ruby/object:Api::Type::String
```

Comment:
are you planning to add sha1_certs and sha256_certs in a followup PR?

Reply:
Yes, these fields will be added in a subsequent PR.